indico/modules/rb_new/controllers/backend/bookings.py

```diff
@@ -387,4 +391,5 @@ def _process(self):
             self.reject()
         elif self.action == 'cancel':
             self.occurrence.cancel(session.user)
+        signals.rb.booking_occurrence_state_changed.send(self.occurrence)
```

Comment:
Shouldn't this be done in the cancel/reject methods of `ReservationOccurrence`? Otherwise it won't be triggered when something else cancels it (e.g. a colliding booking that has priority)

Reply:
Hm, I wasn't sure about this. Initially it was inside the corresponding methods and I moved it here at some point because of some reason (don't remember now what the reason was :P)